hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -249,9 +249,11 @@ public Response get(
 
     AUDIT.logReadSuccess(buildAuditMessageForSuccess(s3GAction,
         getAuditParameters()));
+    int keyCount =
+        response.getCommonPrefixes().size() + response.getContents().size();
     getMetrics().updateGetBucketSuccessStats(startNanos);
-    response.setKeyCount(
-        response.getCommonPrefixes().size() + response.getContents().size());
+    getMetrics().incListKeyCount(keyCount);
```

Comment:
Is the `listKeyCount` metric really intended to be increased by the number of keys?  I thought it would be the numnber of operations, in which case it should be incremented by 1.

Reply:
This metric is used to measure the list request load, recording the number of keys returned by a list requests. 
The time taken for a list request to return 1000 keys must be different from that for 10 keys, using the list count Metric (`incListKeyCount()`) and the list time consumption Metric (`updateGetBucketSuccessStats()`) can better locate performance issue.

Reply:
I see, thanks for the explanation.